Assets/Editor/MenuItems.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Interactables;
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor
+{
+    /// <summary>
+    /// Manager for editor menu items, etc.
+    /// </summary>
+    public class MenuItems : MonoBehaviour
```

Comment:
I don't believe these have to be MonoBehaviours

in fact if they are and someone accidentally adds this, it'll probably break a Standalone build